modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -0,0 +1,44 @@
+
+== Store formats in Neo4j
+Neo4j has several defined store formats, each with a set of different features and limitations. The store format of a database is decided when the database is created, most commonly controlled by the `db.format` setting in the neo4j.conf file, or as an argument to the tool creating the database, e.g `import` or `copy` tools. The current available formats are
+
+* `aligned`
+* `standard`
+* `high_limit` label:enterprise-edition[]
+* `block` label:enterprise-edition[]
+
+== Properties of a store formats
+The store format is responsible for how data is written to disk and how to read it. Some key aspects that may differ between formats
+
+* The performance and resource consumption of read and write transactions. An operation may be faster on one store format than another.
+* The physical size on disk. Two databases with identical logical data on different format may have different size.
+* Performance in memory constrained environments. The same query on different formats may have different page cache hit ratio when the database does not fully fit in the page cache
+* The amount of data that can be stored. See xref:tools/neo4j-admin/neo4j-admin-store-info.adoc#neo4j-admin-store-entity-limits[entity limits] for limitations of the individual formats.
+* The order of query results when the order is not specified. Different store formats may traverse and return data in different order
+* Algorithms used by admin tools. Especially `check` and `import` commands. Performance and resource utilization may differ 
+
+
+== Changing store format
+Each format has a different layout of the data stored on disk. Changing the format is possible to do by re-writing all data in the new format, which only can be done when the database is offline. The `migrate` and `copy` tools provide this functionality.
+
+
+[NOTE]
+====
+Changing format of a database requires enough free disk space to hold both the current and the new database. It may also be a time consuming operation, depending on the size and complexity of the data in the database. The performance of migration is heavily dependent on the speed of the disk and the amount of available memory
+====
+
+[WARNING]
+====
+Different formats have different limitations, e.g number of nodes and relationships that can at most fit. Ensure that the new format can fit all the data in the database.
+====
+
+== Standard, Aligned and High-limit formats
+These formats stores most graph data in linked-list like structures on disk. The high-limit format allows for more nodes and and relationships to be stored, at the cost of slightly lower performance and bigger disk size. 
+
+
+== Block format
+The block format is utilizing multiple different data structures to allow for data locality, i.e. graph data that belongs together is stored in the same place on disk. This allows more related data to be fetched by fewer read operations, resulting in better resource utilization.
```

Comment:
```suggestion
The block format stores related graph data together on disk by using multiple data structures.
This allows more related data to be fetched by fewer read operations, resulting in better resource utilization.
```

Reply:
I think this change removes much of the information. Maybe something like this?
> The block format uses a range of different data structures and inlining techniques to achieve data locality and store related data together on disk

Reply:
sounds good